js/views/ListingCard.js

```diff
@@ -166,7 +166,9 @@ export default class extends baseVw {
             model: this.fullListing,
             profile: this.options.profile,
             vendor: this.options.vendor,
-            closeButtonClass: 'cornerTR ion-ios-close-empty iconBtn clrP clrBr clrSh3',
+            closeButtonClass: 'cornerTR iconBtn clrP clrBr clrSh3 toolTipNoWrap',
+            innerButtonClass: 'ion-ios-close-empty',
+            closeButtonTip: app.polyglot.t('pageNav.toolTip.close'),
```

Comment:
I fixed the issues with the listing edit close button by moving the icon to an inner element, and separating the class for it. The tooltip text is also passed in here, in case a future modal needs different text.